optimum/exporters/onnx/model_configs.py

```diff
@@ -0,0 +1,186 @@
+# coding=utf-8
```

Comment:
So the goal is to centralize the ONNX configs of all models in this file right?

Reply:
It will become a mess at some point XD
But we can think about it later and do this in another PR :+1: